pkgconfig/mbedcrypto.pc.in

```diff
@@ -0,0 +1,9 @@
+includedir=@PKGCONFIG_INCLUDEDIR@
+libdir=@PKGCONFIG_LIBDIR@
```

Comment:
Defining `prefix=` is still needed, please have a closer look at the example I gave you at <https://github.com/simdjson/simdjson/pull/1767> ;)

Reply:
> Defining `prefix=` is still needed, please have a closer look at the example I gave you at [simdjson/simdjson#1767](https://github.com/simdjson/simdjson/pull/1767) ;)

Is that actually true?

Does anything actually run `pkg-config --variable=prefix` ? The man [page](https://linux.die.net/man/1/pkg-config) states most packages define it, but doesn't seem to require it. The PC files that iv'e reviewed for popular packages define it as a hardcoded prefix and then reference the variable within the PC file. Its use in that scenario seems justifiable, but with no internal references to prefix within the package file it seems like it can be omitted. Whats the actual use besides someone querying the variable directly vi apkg-config?

Reply:
It isn't for basic use cases, but having `includedir` reference `prefix` instead of hardcoding the full path makes it possible for users to override the `prefix` variable to make the pkg-config file work even if moved around (i.e. making it somewhat relocatable). It's also a nice convention that's nice to follow, who knows who relies on it for what :)

Reply:
Yeah but in order to have prefix work as intended I would have to inject `${prefix}` in from the CMake side. The current join script in this patch does not do that.

Reply:
> Yeah but in order to have prefix work as intended I would have to inject `${prefix}` in from the CMake side. The current join script in this patch does not do that.

I dropped the follow up patch, if they don't want to take Jan Tojnar's Cmake script and use the CMake builtin in 3.20, I'll have to figure out how to re-roll that because AFAICT the CMake builtin won't let me pass that through when I quickly tried